pyproject.toml

```diff
@@ -52,11 +52,11 @@ Issues = "https://github.com/pelme/htpy/issues"
 html2htpy = "htpy.html2htpy:main"
 
 [build-system]
-requires = ["flit_core >=3.2,<4"]
-build-backend = "flit_core.buildapi"
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
```

Comment:
Did you consider setuptools+setuptools-scm instead of hatchling?

I am playing with a rust implementation of htpy and if that turns out to be a good idea/useful, I think using setuptools+setuptools-rust+setuptools-scm would be the most sensible combination. Currently I have been using maturin for my testing but if I were to actually merge it, I think setuptools-rust would be nicer. IIUC maturin cannot do the automatic VCS part. So if there is no particular reason for hatchling over setuptools, maybe we could just go with setuptools directly. :)

Reply:
I actually looked briefly at maturin after making this PR, and was a bit worried that this change would conflict with maturin. Didn't know about setuptools-rust!

I've updated the PR now to use setuptools + setuptools-scm instead.